sparsimony/pruners/unstructured.py

```diff
@@ -55,16 +55,18 @@ def calculate_mask(
         mask: torch.Tensor,
         weights: torch.Tensor,
     ) -> torch.Tensor:
-        n_drop = int(mask.sum(dtype=torch.int) * prune_ratio)
+        n_drop = int(mask.sum() * prune_ratio)
         scores = torch.where(
             mask == 1, torch.abs(weights), torch.full_like(weights, np.inf)
         )
         if dist.is_initialized():
             dist.all_reduce(scores, dist.ReduceOp.AVG, async_op=False)
-        _, indices = torch.topk(scores.view(-1), k=n_drop, largest=False)
+        _, indices = torch.topk(scores.reshape(-1), k=n_drop, largest=False)
```

Comment:
Keep as view, reshape has higher overhead
```suggestion
        _, indices = torch.topk(scores.view(-1), k=n_drop, largest=False)
```